plugins/woocommerce-blocks/assets/js/blocks/product-collection/edit/inspector-controls/linked-product-control.tsx

```diff
@@ -136,35 +147,40 @@ const LinkedProductControl = ( {
 	location: WooCommerceBlockLocation;
 	usesReference: string[] | undefined;
 } ) => {
-	const REFERENCE_TYPE_PRODUCT = 'product';
-	const isProductReferenceAvailable =
-		location.type === REFERENCE_TYPE_PRODUCT;
+	const isProductLocation = location.type === 'product';
+	const isUsesReferenceIncludesProduct =
```

Comment:
Totally nitpicky; can we rename this const to something like: `hasProductReference`?
I just want something smaller and more descriptive.

Reply:
I don't have a strong opinion on this, so made the changes in d6de3a2